exporter/exporterhelper/internal/persistent_storage.go

```diff
@@ -210,17 +183,16 @@ func (pcs *persistentContiguousStorage) put(req Request) error {
 
 	reqBuf, err := pcs.marshaler(req)
 	if err != nil {
-		return err
+		return false
```

Comment:
Why do we need to suppress the error here? I know that Produce eventually does that, but I don't think that's the right behavior. I think we should propagate the error to the queue sender

Reply:
That is a valid approach and maybe we should do that. this was just trying to consolidate on one interface.